development

amrabdalsattar opened the pull request.

amrabdalsattar committed:
- Maintain the Splash Screen Issue
- Added Dialog Utils
- Completed the Authentication Architecture and UI

amrabdalsattar committed:
- Upgrade Flutter SDK and Replace Deprecated Codes

amrabdalsattar committed:
- Implement Relative imports

amrabdalsattar committed:
- Implement DI
- Establish Shared Preferences Utilities
- Cached User's data when he/she Logged in or Registered

amrabdalsattar committed:
- The Authentication is Finished

amrabdalsattar committed:
- Changed Canvas Color to primary Color
- Added the tabs view
- Added Bottom Navigation Bar
- Added Main Screen View Model

amrabdalsattar committed:
- Added the Api Factory

hossamgamal-1 reviewed (changes requested):
Pull request has an ambiguous name and has no discription.

amrabdalsattar committed:
- Added Product DataModel
- Finished Categories and Products Networking
- Finished Categories UI
- Replaced Connectivity by InternetConnectionChecker
